### DISCUSSION TITLE
Cookie Icon

### DISCUSSION THREAD

**Discussion by nick-0101** [tailwindlabs/heroicons#373] [Ideas]
Pretty much every website requests to use cookies. It would be nice to have a cookie icon to go along with the popup/banner.

**Comment by pixelass** [comment]
Is there any plan to include this at some point. It seems so awkward that this is not part of the initial set.

**Reply by nick-0101** [reply]
3 years later and it's still not added, I wouldn't expect it anytime soon. I've moved onto other icon libraries like [tabler icons](https://tabler.io/icons)

**Comment by ndri** [comment]
I added `cookie` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=cookie
